lib/studies/reply/adaptive_nav.dart

```diff
@@ -909,63 +916,99 @@ class _ReplyLogo extends StatelessWidget {
   }
 }
 
-class _ReplyFab extends StatelessWidget {
-  const _ReplyFab({this.extended, this.onMailView});
+class _ReplyFab extends StatefulWidget {
```

Comment:
I think this can be a `StatelessWidget`

Reply:
`AnimatedSize` requires a ticker for the animation, so I had to make it `Stateful`.

Reply:
Ahh yes!